src/js/utils.js

```diff
@@ -80,13 +80,12 @@ export function isPatientValid(patient) {
 }
 
 function isEmailValid(email) {
-  return email.match(/[a-z0-9!#$%&'*+/=?^_`{|}~.-]+@[a-z0-9-]+(\.[a-z0-9-]+)*/)
+  return email && email.match(/[a-z0-9!#$%&'*+/=?^_`{|}~.-]+@[a-z0-9-]+(\.[a-z0-9-]+)*/)
```

Comment:
Why do you need to check the existance of `email` here before returning and why is the structure so difference than `isIbanValid()`?
I guess returning a boolean would make more sense as well

Reply:
At some point while testing, an error was popping in the console because the patient didn't have any email, so `email` was undefined and `email.match` was throwing an error. Even if, at the end, all patients have an email, I added this check to prevent any error.
You're right about IBAN. I think it is code evolution that led to this difference. It's now fixed and homogenized. For RCC validation too.